reminders.go

```diff
@@ -29,6 +34,31 @@ func (api *Client) doReminder(ctx context.Context, path string, values url.Value
 	return &response.Reminder, response.Err()
 }
 
+func (api *Client) doReminders(ctx context.Context, path string, values url.Values) ([]*Reminder, error) {
+	response := &remindersResp{}
+	if err := api.postMethod(ctx, path, values, response); err != nil {
+		return nil, err
+	}
+
+	// create an array of pointers to reminders
+	var reminders = make([]*Reminder, 0)
```

Comment:
You can get the length of slice, so it's best to allocate it in advance.

```suggestion
	var reminders = make([]*Reminder, 0, len(response.Reminders)
```